### ISSUE TITLE
 Mechanical extraction of helium 

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1159]
```
Application Version: 5.5.1 Serial
SVN Revision Number:6385
Compile Date: Tue, 22 Jun 2010
Operating System: Windows XP Version 2002


Describe details of the issue below:
************************************

I have some strange results while trying to model the mechanical
extraction of Helium in a compartment.

I have a quite simple case composed of:
-       2 sources of Helium with a MASS_FLUX of 0.5 kg/m2.s
-       A “smoke exhaust” with a VOLUME_FLUX of 25 m3/s.

When I run my case, a helium layer appears due to density differences
and buoyancy effects. The helium layer height is approximately
constant after 15seconds.

I have run this case with 2 versions of FDS5 and plotted a slice file
of density in Smokeview.

-        With FDS 5.5.1, the density close to the “smoke exhaust” is
lower than 0.3 while the density in the rest of the layer is between
0.7 and 1.2. It looks like the “smoke exhaust” filters the helium and
extracts only the air… This does not seem to be very "physical".
-        With FDS 5.4.3, everything looks ok. The gas close to the
“smoke exhaust” has the same density than the gas layer around the
“smoke exhaust”.

Since it is not easy for me to explain this problem clearly, I have
uploaded pictures and fds file (Vent_Helium_Issue.zip).

Does anybody know why such different results are obtained? What do you
think about the physics of the case? Do I have some wrong input in my
FDS file which could explain such results? 


Details of the FDS 5.4.3 version:
*********************************

Application Version: 5.4.3 Serial
SVN Revision Number:5210
Compile Date: Thu, 03 Dec 2009
Operating System: Windows XP Version 2002


```

Original issue reported on code.google.com by `mathieu.suzanne` on 2010-08-30 14:57:34

<hr>
- _Attachment: [Vent_Helium_Issue.zip](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1186/comment-0/Vent_Helium_Issue.zip)_

**Comment by gforney** [comment]
```
I'll take a look at it.
```

Original issue reported on code.google.com by `mcgratta` on 2010-08-30 15:05:40

**Comment by gforney** [comment]
```
There are several problems in the input file:

1. Missing / on SURF line for 'EXAUST FAN'.
2. AIR should not be explicitly declared a SPECies.
3. No need to define INERT as DEFAULT.
```

Original issue reported on code.google.com by `mcgratta` on 2010-08-30 15:30:37

**Comment by gforney** [comment]
```
4. ISOTHERMAL=.TRUE. but the BRICK is given thermal properties. This may not be a mistake,
but I worry that the inconsistent input could lead to a problem.
```

Original issue reported on code.google.com by `mcgratta` on 2010-08-30 15:35:48

**Comment by gforney** [comment]
```
1. I am a stupid scatterbrain...

2. You're right... I have defined AIR as a SPECies because I wanted to inject a mixture
of AIR and HELIUM later on... But I should have defined a BACKGROUND_SPECIES different
from AIR...

4. I also agree for thermal properties of BRICK. At the beginning I used ISOTHERMAL=.FALSE.
(as default) but, in that case, when I run my fds file I obtained temperature lower
than 9C... (please, see the attached file). I thought it could be a consequence of
the way the temperature is calculated using the density and background pressure (equation
4.5 of technical guide). Specifying ISOTHERMAL = .TRUE. fix this temperature "issue"
and that's why I stop wandering why it appears.
```

Original issue reported on code.google.com by `mathieu.suzanne` on 2010-08-30 16:24:34

<hr>
- _Attachment: test72_0041.png<br>![test72_0041.png](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1186/comment-4/test72_0041.png)_

**Comment by gforney** [comment]
```
Changes that we are making for FDS 6 should eliminate the temperature problem. It is
related to numerical error in regions of high gradients.

Even with your input file issues, I believe that there is still a bug to fix. We will
fix and hopefully have a new version within the next two weeks.
```

Original issue reported on code.google.com by `mcgratta` on 2010-08-30 17:38:42

**Comment by gforney** [comment]
```
I have fixed the bug. FDS 5.5.2 should be released within the next few weeks.
```

Original issue reported on code.google.com by `mcgratta` on 2010-08-31 13:52:25

**Comment by gforney** [comment]
```
Thank you very much.
```

Original issue reported on code.google.com by `mathieu.suzanne` on 2010-08-31 14:05:57

**Comment by gforney** [comment]
```
The bug is not present anymore in the new released version (5.5.2 serial).
```

Original issue reported on code.google.com by `mathieu.suzanne` on 2010-09-09 12:22:35

**Comment by gforney** [comment]
```
Great, thanks for checking on this.
```

Original issue reported on code.google.com by `mcgratta` on 2010-09-09 12:43:48
